scripts/docker/ubuntu.16.10/Dockerfile

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+# Dockerfile that creates a container suitable to build dotnet-cli
+FROM ubuntu:16.10
+
+# Misc Dependencies for build
+RUN apt-get update && \
+    apt-get -qqy install \
+        curl \
+        unzip \
+        gettext \
+        sudo && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# This could become a "microsoft/coreclr" image, since it just installs the dependencies for CoreCLR (and stdlib)
+RUN apt-get update && \
+    apt-get -qqy install \
+        libunwind8 \
+        libkrb5-3 \
+        libicu57 \
+        liblttng-ust0 \
+        libssl1.0.0 \
+        zlib1g \
+        libuuid1 \
+        liblldb-3.5 && \
```

Comment:
There is actually another debhelper command that gives an error if this is not installed. We could (potentially, haven't tested) add some more logic to the python script to pass the "ignore flag" to that command, too, but it didn't seem worth it at this point. The other Ubuntu Dockerfiles install liblldb for the same reason, it looks like.